sim/priest/shadow/rotation.go

```diff
@@ -150,6 +156,14 @@ func (spriest *ShadowPriest) tryUseGCD(sim *core.Simulation) {
 			TFmod = 0
 		}
 
+		mfglyphMod := 1.0
+		if spriest.HasGlyph(int32(proto.PriestMajorGlyph_GlyphOfMindFlay)) {
```

Comment:
Instead of casting all of these glyphs, create helper functions for priest, analogous to https://github.com/wowsims/wotlk/blob/master/sim/hunter/hunter.go#L84

Reply:
The helper function seems pretty close to what is already in this code. Small implementation differences, but seems fine. Maybe im not understanding.